### PR TITLE
Temporarily disable the format linter

### PR DESCRIPTION
**Pull request by ymesika**
Disable it temporarily (but keep other linter checks) until https://github.com/golang/go/issues/28200 is fixed.
Included are lint and dep fixes to errors that went in since the lint changed to not required.
Re-enabling the lint test will avoid further non-format errors from piling up while the lint test is not required.

### CODE REVIEW

**Comment by ymesika** [comment]
cc @hklai

**Comment by hzxuzhonghu** [comment]
Since there is no accurate time when https://github.com/golang/go/issues/28200 can be fixed. And during these time, should not introduce other failure. 

After this should make lint required.

/lgtm

**Comment by hklai** [comment]
Looks like there are some license related errors. Could you help resolve them in the same PR?

**Comment by ymesika** [comment]
@hklai The errors were already fixed in #9423

File: galley/pkg/kube/interfaces.go [hunk 1]
```diff
@@ -58,34 +56,34 @@ func (k *kube) DynamicInterface(gv schema.GroupVersion, kind, listKind string) (
...
 
-func addTypeToScheme(s *runtime.Scheme, gv schema.GroupVersion, kind, listKind string) error {
-	builder := runtime.NewSchemeBuilder(func(s *runtime.Scheme) error {
-		// Add the object itself
-		gvk := schema.GroupVersionKind{
-			Group:   gv.Group,
-			Version: gv.Version,
-			Kind:    kind,
-		}
+// func addTypeToScheme(s *runtime.Scheme, gv schema.GroupVersion, kind, listKind string) error {
```
Comment (hklai) [inline galley/pkg/kube/interfaces.go]: Remove instead if unused?

File: galley/pkg/kube/interfaces_test.go [hunk 2]
```diff
@@ -17,25 +17,24 @@ package kube
...
 
-	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/client-go/rest"
 )
 
-func TestAddTypesToScheme(t *testing.T) {
-	gv := schema.GroupVersion{Group: "group", Version: "version"}
-	s := runtime.NewScheme()
+// func TestAddTypesToScheme(t *testing.T) {
```
Comment (hklai) [inline galley/pkg/kube/interfaces_test.go]: same here?

File: pkg/test/framework/components/apps/local/agent/pilot_agent_test.go [hunk 3]
```diff
@@ -274,6 +274,9 @@ func forwardRequestToAgent(a Agent, req *proto.ForwardEchoRequest) ([]*echo.Pars
 	}
 	client, err := echo.NewClient(fmt.Sprintf("127.0.0.1:%d", grpcPortA.ApplicationPort))
 	defer client.Close()
+	if err != nil {
```
Comment (hklai) [inline pkg/test/framework/components/apps/local/agent/pilot_agent_test.go]: this looks like a logic change instead?

**Comment by ymesika** [comment]
@quanjielin Yes, you had dead code referencing that vendor code.
If you put it back can you please make sure the lint test pass?

**Comment by quanjielin** [comment]
I added placeholder in security/pkg/nodeagent/plugin/providers/google/iamclient.go refers to that pkg, which you comment it out...

is there any lint test fail ? https://github.com/istio/istio/pull/9420 presubmit pass for me.

**Comment by ymesika** [comment]
Yes, I preferred to comment it out because if you uncomment the CircleCI `lint` test will fail gomeatlinter complaining about deadcode.

You can uncomment my comments, add the vendor and see if the lint test pass.

**Comment by quanjielin** [comment]
by running lint test you mean ```make lint```  ?

**Comment by ymesika** [comment]
Yep. Or just run `bin/linters.sh`. Same.

**Pull request merged**